packages/protocol/contracts/governance/test/SlashingTestUtils.sol

```diff
@@ -0,0 +1,73 @@
+pragma solidity ^0.5.3;
+
+import "../DoubleSigningSlasher.sol";
+
+contract SlashingTestUtils {
```

Comment:
Consider "MockUsingPrecompiles" (if you think that's an accurate name) for consistency